examples/problem_decomposition/mifno.ipynb

```diff
@@ -152,30 +152,35 @@
     "mi_solver = IncrementalDecomposition(solver=fno, truncation_order=3)\n",
     "\n",
     "# Submitting the problem to the cloud.\n",
-    "HF_handle = mi_solver.simulate(molecule=HF_mol)\n",
-    "print(f\"\\nHF handle: {HF_handle}\")\n",
+    "BeH2_handle = mi_solver.simulate(molecule=BeH2_mol)\n",
+    "print(f\"\\nBeH2 handle: {BeH2_handle}\")\n",
     "\n",
     "# Retrieving the results.\n",
-    "HF_result = get_results(HF_handle)\n",
-    "print(HF_result)\n",
+    "BeH2_result = get_results(BeH2_handle)\n",
+    "print(BeH2_result)\n",
     "```\n",
     "\n",
-    "The `HF_result` python dictionary can be saved in a json file (for further treatment in Tangelo or for archiving purposes) with the following code.\n",
+    "With the problem handle, one can download the log file, where all the information can be retrieved for further treatment in Tangelo or for archiving purposes. This is done in the next cell with the following code. Due to a bug in QEMIST Cloud, the urls to get the molecular orbitals coefficients for each fragment must be retrieved from the `get_problems_url` function, and not directly from the log file. \n",
```

Comment:
The second part of the sentence is not good. Just tell people how it can be done now, and indicate if that may change and how later.